gcd/src/main/java/org/spine3/server/storage/datastore/DsPropertyStorage.java

```diff
@@ -40,12 +40,12 @@
  */
 /* package */ class DsPropertyStorage {
 
-    private static final String PROPERTIES_KIND = "properties_kind";
-    private static final String ANY_TYPE_URL = TypeName.of(Any.getDescriptor()).toTypeUrl();
+    private static final TypeUrl ANY_TYPE_URL = TypeUrl.of(Any.getDescriptor());
 
     private final DatastoreWrapper datastore;
 
-    /* package */ static DsPropertyStorage newInstance(DatastoreWrapper datastore) {
+    /* package */
```

Comment:
please put to same line. You can use `/\* package \*/\n` such search regExp pattern to find all such situations.